docs/resources/css/dark.css

```
@@ -0,0 +1,1996 @@
+        color: rgb(190, 103, 215);
+    }
+
+    .highlight .il {
+        color: rgb(125, 222, 174);
+    }
+
+    .rst-other-versions a {
+        border-color: initial;
+    }
+
+    .ethical-sidebar .ethical-image-link,
+    .ethical-footer .ethical-image-link {
+        border-color: initial;
+    }
+
+    .ethical-sidebar,
+    .ethical-footer {
+        background-color: rgb(34, 36, 38);
+        border-color: rgb(62, 68, 70);
+        color: rgb(226, 223, 219);
+    }
+
+    .ethical-sidebar ul {
+        list-style-image: initial;
+    }
+
+    .ethical-sidebar ul li {
+        background-color: rgb(5, 77, 121);
+        color: rgb(232, 230, 227);
+    }
+
+    .ethical-sidebar a,
+    .ethical-sidebar a:visited,
+    .ethical-sidebar a:hover,
+    .ethical-sidebar a:active,
+    .ethical-footer a,
+    .ethical-footer a:visited,
+    .ethical-footer a:hover,
+    .ethical-footer a:active {
+        color: rgb(226, 223, 219);
+        text-decoration-color: initial !important;
+        border-bottom-color: initial !important;
+    }
+
+    .ethical-callout a {
+        color: rgb(161, 153, 141) !important;
+        text-decoration-color: initial !important;
+    }
+
+    .ethical-fixedfooter {
+        background-color: rgb(34, 36, 38);
+        border-top-color: rgb(66, 72, 74);
+        color: rgb(192, 186, 178);
+    }
+
+    .ethical-fixedfooter .ethical-text::before {
+        background-color: rgb(61, 140, 64);
+        color: rgb(232, 230, 227);
+    }
+
+    .ethical-fixedfooter .ethical-callout {
+        color: rgb(168, 160, 149);
+    }
+
+    .ethical-fixedfooter a,
+    .ethical-fixedfooter a:hover,
+    .ethical-fixedfooter a:active,
+    .ethical-fixedfooter a:visited {
+        color: rgb(192, 186, 178);
+        text-decoration-color: initial;
+    }
+
+    .ethical-rtd .ethical-sidebar {
+        color: rgb(184, 178, 169);
+    }
+
+    .ethical-alabaster a.ethical-image-link {
+        border-color: initial !important;
+    }
+
+    .ethical-dark-theme .ethical-sidebar {
+        background-color: rgb(58, 62, 65);
+        border-color: rgb(75, 81, 84);
+        color: rgb(193, 188, 180) !important;
+    }
+
+    .ethical-dark-theme a,
+    .ethical-dark-theme a:visited {
+        color: rgb(216, 213, 208) !important;
+        border-bottom-color: initial !important;
+    }
+
+    .ethical-dark-theme .ethical-callout a {
+        color: rgb(184, 178, 169) !important;
+    }
+
+    .keep-us-sustainable {
+        border-color: rgb(87, 133, 38);
+    }
+
+    .keep-us-sustainable a,
+    .keep-us-sustainable a:hover,
+    .keep-us-sustainable a:visited {
+        text-decoration-color: initial;
+    }
+
+    .wy-body-for-nav .keep-us-sustainable {
+        color: rgb(184, 178, 169);
+    }
+
+    .wy-body-for-nav .keep-us-sustainable a {
+        color: rgb(222, 219, 215);
+    }
+
+    /* For black-on-white/transparent images at handbook/text-anchors.html */
+    #text-anchors img {
+        filter: invert(1);
```

I think it's a bit too bright, certainly more than the rest of the page. I think the following is a good balance:
```suggestion
        filter: invert(1) brightness(0.85);
```